src/cpp/desktop/DesktopWin32ApplicationLaunch.cpp

```diff
@@ -151,12 +154,7 @@ bool ApplicationLaunch::sendMessage(QString filename)
                                                        0));
       if (::IsWindow(hwnd))
       {
-         HWND hwndPopup = ::GetLastActivePopup(hwnd);
```

Comment:
This code has to deal with activating an existing instance (e.g. when a file association is used to open a file in an existing instance of the IDE). So you should check to see whether this still works after the change (it's also possible that other aspects of the QtWebEngine migration could have broken this)

Reply:
This does still work after the change. Note that the code here was replaced with a single call to `activate(hwnd)` as the inline code here was identical to `activate()`.